steps/generate_dataset_from_htme.py

```diff
@@ -440,7 +440,7 @@ def log_end_of_batch(correlation_id, run_id, status, dynamodb=None):
     start_time = time.perf_counter()
     dynamodb = get_resource('dynamodb')
     run_id = log_start_of_batch(args.correlation_id, dynamodb)
-    main(spark, s3_client, s3_htme_bucket, s3_prefix, secrets_collections, keys_map,
+    main(spark, s3_client, s3_htme_bucket, args.s3_prefix, secrets_collections, keys_map,
```

Comment:
Please refactor to use s3_prefix from args itself and delete this `args.s3_prefix` from the list of arguments. I think we are already passing `args` as the method argument for correlation_id so we can do the same for s3_prefix as well